boards/nucleo-l1/include/periph_conf.h

```diff
@@ -213,6 +213,46 @@ static const timer_conf_t timer_config[] = {
 #define GPIO_15_IRQ         EXTI15_10_IRQn
 
 /**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_NUMOF           (2U)
+#define PWM_0_EN            1
+#define PWM_1_EN            1 
+#define PWM_MAX_CHANNELS    4
+
+/* PWM 0 device configuration */
+#define PWM_0_DEV           TIM3
+#define PWM_0_CHANNELS      4
+#define PWM_0_CLK           (168000000U)
```

Comment:
I doubt the source clock of the PWM runs at 168MHz ;) Guess this is copied from the F4.